core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.ts

```diff
@@ -29,15 +29,19 @@ import { AppConstants } from 'app.constants';
   templateUrl: './contributor-dashboard-admin-navbar.component.html',
 })
 export class ContributorDashboardAdminNavbarComponent implements OnInit {
-  profilePictureDataUrl: string;
-  username: string;
-  profileUrl: string;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion. For more information, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  profilePictureDataUrl!: string;
+  // User name is null if the user is not logged in.
+  username!: string | null;
```

Comment:
This seems strange to me as we are using a non-null assertion operator, yet saying that the value can also be null. Maybe just don't assign the variable until after checking the value of `userInfo.getUsername()`.

Reply:
Done, Actually forget to assign null directly.
Thanks.